vassal-app/src/main/java/VASSAL/launch/ModuleManager.java

```diff
@@ -38,6 +38,7 @@
 
 import javax.swing.SwingUtilities;
 
+import VASSAL.configure.BooleanConfigurer;
```

Comment:
Goes with our other imports!

Reply:
Haha! Lest you think I've got something weird wrong with my brain when it comes to imports, it's my IDE that auto-puts these in sometimes-weird-places. I often don't even realize I've added an import to the file because it just kind of automatically does it as I reference new things.

Anyway, all fixed.

Reply:
That's weird that it doesn't pay attention to grouping.